src/webapi/html_elements/input.rs

```diff
@@ -38,4 +38,44 @@ impl InputElement {
             @{self}.value = @{value};
         }
     }
+
+    /// The offset to the start of the selection.
```

Comment:
Add an extra newline (a line with only `///`) after summary. (This also applies to other methods you've added.)